src/omnicore/doc/release-notes.md

```diff
@@ -1,7 +1,7 @@
-Omni Core v0.0.11
-=================
+Omni Core v0.0.11.1
+===================
 
-v0.0.11 is a major release and consensus critical in terms of the Omni Layer protocol rules. An upgrade is mandatory, and highly recommended. Prior releases will not be compatible with new behavior in this release.
+v0.0.11.1 is a major release and consensus critical in terms of the Omni Layer protocol rules. An upgrade is mandatory, and highly recommended. Prior releases will not be compatible with new behavior in this release.
 
```

Comment:
Updated. Still looks a bit strange, due to the `v0.0.11.1 is ... v0.0.11 is a major ...`, but it's much clearer now.